### PR TITLE
🧹 refactor: unify feature flag on/off confirm messages

### LINKED ISSUE #333: [Good First Issue]: Unify Feature Flag On/Off Confirm Messages
# Description

We want to unify the turn feature flag on/off confirm messages for better user experience. Below is the messages we proposed.

## Turn Flag On Confirm Message
- `i18n-nz-popconfirm="@@ff.are-you-sure-to-turn-ff-on"`
- `nz-popconfirm="Are you sure to turn it on? The flag will return the matching targeting users' or rules' serving variation."`
- Chinese translation: 确定要打开开关么？打开后开关将按照既定规则匹配用户返回值。

## Turn flag Off Confirm Message
- `i18n-nz-popconfirm="@@ff.are-you-sure-to-turn-ff-off"`
- `nz-popconfirm="Are you sure to turn it off? The flag will return the variation that you specified for its off state."`
- Chinese translation: 确定要关闭开关么？关闭后开关将返回处于关闭状态时的设定值。

# Scope

- Service: UI
- Components: feature flag index and settings components
- The files need to be changed: 
   - modules/front-end/src/app/features/safe/feature-flags/index/index.component.html. Line 56-57
   - modules/front-end/src/app/features/safe/feature-flags/details/setting/setting.component.html. Line 97-98 104-105

# Screenshot

## feature flag index component
image

## feature flag settings component
image

# Setup dev environment

Please read the [doc](../blob/main/Development.md) to set up your development environment.

# Deadline

We usually give 2 weeks for a feature request, which means you have 2 weeks to implement the feature starting from the day it is assigned to you. If you have any difficulties, please contact us in [Slack](https://join.slack.com/t/featbit/shared_invite/zt-1ew5e2vbb-x6Apan1xZOaYMnFzqZkGNQ).

# How to implement

1. Fork the FeatBit repository.
2. Create a branch issues-{issues number}
3. Implement the task based on Description and Scope mentioned above.
4. Submit your PR

Make sure you complete the following tasks before submit your PR:
- merge featbit main branch into your working branch
- run `npm run i18n`
- commit **messages.xlf & messages.zh.xlf** file changes (if any)

# Contribution points

Each contribution counts a certain number of points, depending on its difficulty. Contributors earn the appropriate
points when the work is merged. Contribution points are used to describe the contributions that contributors have made.
They can also be used to receive rewards for community events. You can view the current community submissions on a
public [google sheet](https://docs.google.com/spreadsheets/d/1ukyXgi_jRPeXj7EAST0IrnPfLOQ6xDBkcyAJY9N-Yb4/edit#gid=1117970540).

Points: 4

# How to claim to solve the issue

If you want to implement this function, please leave a comment in this issue like:

> I'd like to implement this function, please assign this issue to me.

### PR DESCRIPTION
**Pull request by Shubh8899**
Resolves #333 Unify Feature Flag On/Off Confirm Messages

### CODE REVIEW

**Comment by cosmos-explorer** [comment]
@all-contributors
please add @Shubh8899 for code

File: modules/front-end/src/app/features/safe/feature-flags/index/index.component.html [hunk 1]
```diff
@@ -95,14 +95,14 @@
...
             <nz-switch *ngIf="featureFlag.isEnabled"
                        i18n-nz-popconfirm="@@ff.idx.are-you-sure-to-disable-ff"
-                       nz-popconfirm="Are you sure to turn it off? The variation returned would be changed when it is off"
+                       nz-popconfirm="Are you sure to turn it off? The flag will return the variation that you specified for its off state."
                        (nzOnConfirm)="onToggleFeatureFlagStatus(featureFlag)"
                        nzControl="true"
                        [ngModel]="featureFlag.isEnabled" nzCheckedChildren="ON" nzUnCheckedChildren="OFF">
             </nz-switch>
             <nz-switch *ngIf="!featureFlag.isEnabled"
                        i18n-nz-popconfirm="@@ff.idx.are-you-sure-to-enable-ff"
```
Comment (deleteLater) [inline modules/front-end/src/app/features/safe/feature-flags/index/index.component.html]: Use the unified i18n: `i18n-nz-popconfirm="@@ff.are-you-sure-to-turn-ff-on`

File: modules/front-end/src/app/features/safe/feature-flags/index/index.component.html [hunk 2]
```diff
@@ -95,14 +95,14 @@
           <td>
             <nz-switch *ngIf="featureFlag.isEnabled"
                        i18n-nz-popconfirm="@@ff.idx.are-you-sure-to-disable-ff"
```
Comment (deleteLater) [inline modules/front-end/src/app/features/safe/feature-flags/index/index.component.html]: Use the unified i18n: `i18n-nz-popconfirm="@@ff.are-you-sure-to-turn-ff-off`

**Comment by deleteLater** [comment]
Hi @Shubh8899 , thanks for your work on this PR 👍 I've reviewed your code and added some comments. 

After resolving these comments, you need to update the [i18n](https://angular.io/guide/i18n-overview) files. To do this, follow these steps:

- merge featbit main branch into your working branch
- run `npm run i18n`
- commit messages.xlf & messages.zh.xlf file changes

**Comment by Shubh8899** [comment]
@deleteLater  I have pushed requested changes, please review.

**Review by deleteLater — APPROVED** [review]
LGTM! Thanks for your time and effort ❤️  @Shubh8899

**Comment by deleteLater** [comment]
@all-contributors
please add @Shubh8899 for code

**Pull request merged**